core/coreapi/interface/unixfs.go

```diff
@@ -29,11 +35,7 @@ type UnixfsAPI interface {
 	//
 	// Note that some implementations of this API may apply the specified context
 	// to operations performed on the returned file
-	Get(context.Context, Path) (files.File, error)
-
-	// Cat returns a reader for the file
-	// TODO: Remove in favour of Get (if we use Get on a file we still have reader directly, so..)
-	Cat(context.Context, Path) (Reader, error)
```

Comment:
We can probably get away with this but we'll need to be smart about whether we optimize for bandwidth or latency. That is, when calling `Cat`, we clearly care about reading the file in-order (latency to next byte). When calling `Get`, we're indicating we just want the entire file (maximize throughput overall).

We can still do this even if we remove the `Cat` function. However, we'll probably need to add some kind of option (or some method on the `UnixfsFile` interface).

We can handle this later, it's just something to keep in mind.

Reply:
Good point, I'd say this is easy enough to handle later